diff --git a/src/Plugin/ActionLink/AJAXactionLink.php b/src/Plugin/ActionLink/AJAXactionLink.php
index 16b0f2e..58c844d 100644
--- a/src/Plugin/ActionLink/AJAXactionLink.php
+++ b/src/Plugin/ActionLink/AJAXactionLink.php
@@ -70,7 +70,7 @@ class AJAXactionLink extends Reload {
       // no matter how many times the flag is clicked.
       return $destination;
     }
-    parent::getDestination();
+    return parent::getDestination();
   }
 
   /**
